src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/RobotString.java

```diff
@@ -0,0 +1,37 @@
+package com.github.markusbernhardt.seleniumlibrary.keywords;
+
+import com.github.markusbernhardt.seleniumlibrary.RunOnFailureKeywordsAdapter;
+import org.apache.commons.lang3.StringUtils;
+import org.robotframework.javalib.annotation.ArgumentNames;
+import org.robotframework.javalib.annotation.Autowired;
+import org.robotframework.javalib.annotation.RobotKeyword;
+import org.robotframework.javalib.annotation.RobotKeywords;
+
+@RobotKeywords
+public class RobotString extends RunOnFailureKeywordsAdapter {
+
+    /**
+     * Instantiated Logging keyword bean
+     */
+    @Autowired
+    protected Logging logging;
+
+    @RobotKeyword("Set System Property")
+    @ArgumentNames({"key", "value"})
+    public void setSystemProperty(String key, String value) {
+        System.setProperty(key, value);
+    }
+
+    @RobotKeyword("Get System Property")
```

Comment:
Maybe some comment how this differs from http://robotframework.org/robotframework/latest/libraries/OperatingSystem.html#Set%20Environment%20Variable would be nice. Or refer to https://www.baeldung.com/java-system-get-property-vs-system-getenv.

Reply:
I agree with you)

Deleted